lib/rspec/core/runner.rb

```diff
@@ -110,6 +110,9 @@ def setup(err, out)
       def run_specs(example_groups)
         @configuration.reporter.report(@world.example_count(example_groups)) do |reporter|
           @configuration.with_suite_hooks do
+            if @world.example_count(example_groups) == 0 && @configuration.fail_if_no_examples
```

Comment:
`@world.example_count(example_groups)` is being called above on line 111 and then again here on line 113.  Instead of calling it twice, can you call it once, store the value in a local variable, and then use it in two places?